src/components/radios/template.njk

```diff
@@ -2,44 +2,46 @@
 {% from "label/macro.njk" import govukLabel %}
 
 {#- innerHtml -> captures the HTML so we can optionally nest it in a fieldset -#}
-<div class="govuk-c-radios
-{%- if params.classes %} {{ params.classes }}{% endif %}"{% for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
-  {%- set innerHtml %}
-  {% for item in params.items %}
-  {% set idPrefix = params.idPrefix if params.idPrefix else params.name %}
-  {% set id = item.id if item.id else idPrefix + "-" + loop.index %}
-  <div class="govuk-c-radios__item">
-    <input class="govuk-c-radios__input" id="{{ id }}" name="{{ params.name }}" type="radio" value="{{ item.value }}"
-    {{-" checked" if item.checked }}
-    {{-" disabled" if item.disabled }}>
-    {{ govukLabel({
-      html: item.html,
-      text: item.text,
-      classes: 'govuk-c-radios__label',
-      attributes: item.label.attributes,
-      for: id
-    }) | indent(4) | trim  }}
-  </div>
-  {% endfor %}
-  {% endset %}
+<div class="govuk-o-form-group {%- if params.errorMessage %} govuk-o-form-group--error{% endif %}">
+  <div class="govuk-c-radios
```

Comment:
The fieldset is nested like

```html
<div class="govuk-o-form-group">
  <div class="govuk-c-radios">
    <div class="govuk-c-fieldset">
    </div>
  </div>
</div>
```

should it be


```html
<div class="govuk-o-form-group">
  <div class="govuk-c-fieldset">
    <div class="govuk-c-radios">
    </div>
  </div>
</div>
```

Reply:
I'd say that's not within the scope of this PR. Are you happy for me to raise an issue?

Reply:
Yeah you're right sorry about that, issue would be good.

Reply:
Here it is: https://github.com/alphagov/govuk-frontend/issues/531